docs/release-procedure.md

```diff
@@ -0,0 +1,209 @@
+# Overview
+
+This document describes the technical procedure for cutting releases.
+Refer to [release.md](release.md) in order to learn about the high-level release process.
+
+# Creating Releases
+
+The actual releases are all cut using the kubevirt release-tool. This tool
+automates the entire process of creating branches, signing tags, generating
+prow configs, and more. All you need to do is gather a few credentials
+in order to use the tool.
+
+## Release Tool Credentials
+
+You must have 2 items before you can create a release.
+
+1. **GPG key and passphrase files used for creating signed releases.**
+
+[Instructions for adding GPG key to your github account](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account)
+
+After adding the GPG key to github, export both the key and passphrase to files.
+Be aware that this results in the key and passphrase being placed into a plain
+text file on your machine. Make sure you don't place this in shared storage.
+
+**Example of exporting key to file**
+
+```gpg --export-secret-key -a <your-email-address> > ${HOME}/gpg-private```
+
+**Example of putting passphrase in file**
+
+```echo "<insert your passphrase here>" > ${HOME}/gpg-passphrase```
+
+2. **Github API token file used for accessing the github api**
+
+When you create this token the only permission you need to give it is the
+ability to access github repositories. That's it.
+
+[Instructions for creating access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
```

Comment:
Here it could be beneficial to note that the classic token should be used.

Reply:
It seems the new one doesn't work for us. At least me and one more person tried and failed.